### PR TITLE
Requires mfg to have memberId when added to network #79

### PR DESCRIPTION
**Pull request by micahu**
#79

### CODE REVIEW

File: contracts/Atonomi.sol [hunk 1]
```diff
@@ -657,6 +657,8 @@ contract Atonomi is Pausable, TokenDestructible {
         m.memberId = _memberId;
 
         if (m.isManufacturer) {
+            require(m.memberId != 0);
```
Comment (filcap) [inline contracts/Atonomi.sol]: lets add a error reason to the require.

**Comment by filcap** [comment]
adding @mraybman for review as well

File: contracts/Atonomi.sol (hunk 1 above)
Comment (filcap) [reply contracts/Atonomi.sol]: also re-run the coverage reports so we can get those refreshed

**Comment by micahu** [comment]
closing, will be pulled in with Fil's next commit

**Pull request closed without merging**